packages/client/lib/sync/sync.ts

```diff
@@ -222,7 +222,7 @@ export abstract class Synchronizer {
     }
     this.clearFetcher()
     clearInterval(this._syncedStatusCheckInterval as NodeJS.Timeout)
-    await new Promise((resolve) => setTimeout(resolve, this.interval))
+    // await new Promise((resolve) => setTimeout(resolve, this.interval))
```

Comment:
Is this no longer needed?  If so, can we just delete it?

Reply:
thanks, thats just something that got introduced while testing some behavior, restored